perf/perf_pcp.py

```diff
@@ -90,10 +90,10 @@ def test_perfevent_24x7_events(self):
         if self.cpu_family == 'power8':
             self.cancel("Not supported on Power8")
         # -s 2 collects two samples
-        cmd = "pmval -s 2 perfevent.hwcounters.hv_24x7.PM_MBA0_CLK_CYC.value"
+        cmd = "pmval -s 2 perfevent.hwcounters.hv_24x7.PM_MBA0_CLK_CYC_chip_0.value"
```

Comment:
just a thought can we maintain this as yaml input

Reply:
As this events may have different values in different systems , so modifying code to fetch events dynamically .